about.html

```diff
@@ -11,58 +11,90 @@
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
+<header>
     
-    Strategy
+    <img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo.">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+<nav>
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
+    <a href="#">Products</a>
+    <a href="#">Blog</a>
+    <a href="#">Content</a>
+</nav>
+</header>
 
+<img class="freeway" src="img/jumbo-about.png" alt="Freeway.">
 
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
+<body>
+    <div class="container about-page">
+     
+  <div class = "about">
+   <h2>About</h2>
 
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </div> <!--about-->
 
+<div class = "packagesof2">
+  <div class = "packages">
+    <img class = "images" src="img/about-plan.png" alt="strategy">
+    
+    <h2>Strategy</h2>
+    
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    <img src="img/about-office.png" alt="strategy">
+    <div class = "learnmore">Learn More</div>
+    
+  </div>
+
+
+  <div class = "packages">
+      <img class = "images" src="img/about-working.png" alt="strategy">
+      
+      <h2>How We Work</h2>
+      
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+  
+      <div class = "learnmore">Learn More</div>
```

Comment:
Let's turn this into a button.

```suggestion
      <button class = "learnmore">Learn More</button>
```